src/main/java/de/srendi/advancedperipherals/common/util/StatusConstants.java

```diff
@@ -0,0 +1,42 @@
+package de.srendi.advancedperipherals.common.util;
+
+/**
+ * A collection of constants used as return types for several peripherals
+ */
+public enum StatusConstants {
+
+    // Crafting Jobs
+    CALCULATION_STARTED,
+    CRAFTING_STARTED,
+    JOB_CANCELED,
+    JOB_DONE,
+    NOT_CRAFTABLE,
+    MISSING_ITEMS,
```

Comment:
Is there any difference between `MISSING_ITEMS` and `ITEM_NOT_FOUND`?

I feel you need explain their usage (if they have different meaning) by comments at here.

Reply:
Described in the new commit
I will completely switch to these new constant in the next PR when this one is merged